src/main.rs

```diff
@@ -434,36 +435,50 @@ fn dice_roll_result_text_ui(
     }
 }
 
+#[derive(Parser, Debug)]
```

Comment:
Nice, I was going to take a swing here next. One change, that env_seed should also default to 0. Going to be using that 0 to mean it's unset, and then generate a secure random version(not going to limit to u64 possibly). This would be helpful for later competitive play(u64 might be a bit small for competitive play with AI, as it's small enough to be 'nearly' brute force trainings).

Reply:
Fixed this one.